applications/openshift/kubelet/kubelet_enable_streaming_connections/tests/notset.fail.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# remediation = none
+
+yum install -y jq
+
+mkdir -p "/etc/kubernetes"
+
+cat << EOF > /etc/kubernetes/kubelet.conf
+{
```

Comment:
This is just a standard kubeconfig that shows this still passes when streamingConnectionIdleTimeout isn't set, right?

Reply:
Yes, that's correct